docs/propertyestimator.rst

```diff
@@ -94,11 +94,11 @@ printed.
                                                                                      # `eTemperature` in specified range
 
     # Load an initial parameter set
-    parameter_set = [ SMIRFFParameterSet('smarty-initial.xml') ]
+    force_field = [ SMIRFFParameterSet('smarty-initial.xml') ]
 
     # Compute physical properties for these measurements
-    estimator = PropertyEstimator(nworkers=10) # NOTE: multiple backends will be supported in the future
-    computed_properties = estimator.computeProperties(dataset, parameter_set)
+    estimator = PropertyEstimatorClient(nworkers=10) # NOTE: multiple backends will be supported in the future
+    computed_properties = estimator.computeProperties(dataset, force_field)
```

Comment:
Can `computeProperties()` really accept an iterable list of of `ForceField` objects?
If so, it would be great if it could also accept a single `ForceField` object so we don't need the awkward syntax of
```python
force_field = [ SMIRFFParameterSet('smarty-initial.xml') ]
computed_properties = estimator.computeProperties(dataset, force_field)
```
and can instead have
```python
force_field = SMIRFFParameterSet('smarty-initial.xml')
computed_properties = estimator.computeProperties(dataset, force_field)
```

Reply:
Unfortunately this is out of date docs again like in the previous comment - they will be updated in the near future.

Reply:
Also, I think the API for this has changed. I see this was from my old notes/sketch!